tests/console/generate_dud.pm

```diff
@@ -13,24 +13,78 @@ use strict;
 use warnings;
 use base "opensusebasetest";
 use testapi;
+use XML::Writer;
 use utils qw(zypper_call);
 use autoyast qw(expand_variables);
 use registration qw(add_suseconnect_product get_addon_fullname);
 
+sub generate_xml {
+    my $addons = get_var('BASE_TEST_REPOS', '');
```

Comment:
there are more settings, not just this one, they all has this format: `*_TEST_REPOS`.